### DISCUSSION TITLE
你们是 Nankai OS 官方吗？

### DISCUSSION THREAD

**Discussion by lazy-forever** [Nankai-OS-2024/OSLabs#1] [Q&A] [answered]
你们是不是 Nankai OS 官方？

**Comment by KaiserTT** [comment, marked as answer]
你说得对，但是 Nankai OS 是一款由 NKU 研发的操作系统课程。在这里，RISC-V 处理器将被授予「ucore」，导引硬件之力。你将扮演「开发者」角色，在编写 ucore 的过程中邂逅惊奇的进程和并发，逐步发掘「虚拟内存」的真相
